commit 9bf4f183c8bae1427c953b274502cc1e2198ab0c
Author: Alexey Kardashevskiy <aik@ozlabs.ru>
Date:   Fri Mar 17 00:48:27 2017 +0000

    powerpc/iommu: Stop using @current in mm_iommu_xxx
    
    [ Upstream commit d7baee6901b34c4895eb78efdbf13a49079d7404 ]
    
    This changes mm_iommu_xxx helpers to take mm_struct as a parameter
    instead of getting it from @current which in some situations may
    not have a valid reference to mm.
    
    This changes helpers to receive @mm and moves all references to @current
    to the caller, including checks for !current and !current->mm;
    checks in mm_iommu_preregistered() are removed as there is no caller
    yet.
    
    This moves the mm_iommu_adjust_locked_vm() call to the caller as
    it receives mm_iommu_table_group_mem_t but it needs mm.
    
    This should cause no behavioral change.
    
    Signed-off-by: Alexey Kardashevskiy <aik@ozlabs.ru>
    Reviewed-by: David Gibson <david@gibson.dropbear.id.au>
    Acked-by: Alex Williamson <alex.williamson@redhat.com>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1435_9bf4f183c8bae1427c953b274502cc1e2198ab0c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1435_9bf4f183c8bae1427c953b274502cc1e2198ab0c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
